app/views/ctc/portal/bank_account/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<% @main_question = t("views.ctc.portal.bank_account.title") %>
+
+<% content_for :page_title, @main_question %>
+
+<% content_for :card do %>
+  <div class="question-layout">
+    <div class="back">
+      <%= link_to t("general.back"), defined?(prev_path) ? prev_path : :back %>
```

Comment:
Consider explicitly defining `prev_path` in the controller and getting rid of the `defined?` check. Not a blocker.